dcpy/connectors/socrata/publish.py

```diff
@@ -69,6 +69,17 @@ class Attachment(TypedDict):
     class Responses:
         """Helper classes to model responses from the Socrata API."""
 
+        class Column(TypedDict):
+            """Column fields returned from the Socrata API."""
+
+            name: str  # ex: Project Name
+            fieldName: str  # ex: projectnam
+            description: str
+            renderTypeName: str
+
+            # cached attributes of a dataset, e.g. top 10 most common values,
+            cachedContents: NotRequired[dict]
```

Comment:
just curious, do you know what this refers to in terms of "cached"? Like, why is something cached in socrata? When do we see cached contents in a response vs not?

Reply:
That's a great question. For MIH, the only column without `cachedContents` is `the_geom`. So maybe certain types aren't cacheable.